[MRG+1] Enable using 2SD scaling in StandardScaler (#10875)
scikit-learn/scikit-learn: doc/modules/preprocessing.rst

**Standardization** of datasets is a **common requirement for many
machine learning estimators** implemented in scikit-learn; they might behave
badly if the individual features do not more or less look like standard
normally distributed data: Gaussian with **zero mean and unit/factor two standard**

> **deviation** in reference to **Gelman, A. (2008). Scaling regression inputs by** 

Reviewer: ```suggestion
**deviation** [Gelman2008]_. 
```

> **dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873**.

Reviewer: ```suggestion
```

Reference
---------

> Gelman, A. (2008). Scaling regression inputs by 

Reviewer: ```suggestion
    .. [Gelman2008] Gelman, A. (2008). Scaling regression inputs by 
```

**Standardization** of datasets is a **common requirement for many
machine learning estimators** implemented in scikit-learn; they might behave
badly if the individual features do not more or less look like standard
normally distributed data: Gaussian with **zero mean and unit/factor two standard**
**deviation** in reference to **Gelman, A. (2008). Scaling regression inputs by** 

> **dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873**.

Reviewer 2: I think you can add a reference bellow the document as

```
.. [1] Gelman, A. (2008). Scaling regression inputs by
dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873
```

and refer to it in the document.

**Standardization** of datasets is a **common requirement for many
machine learning estimators** implemented in scikit-learn; they might behave
badly if the individual features do not more or less look like standard

> normally distributed data: Gaussian with **zero mean and unit/factor two standard**
> **deviation** [Gelman2008]_. 

Reviewer 3: ```suggestion
normally distributed data: Gaussian with **zero mean and unit variance**.
For linear models [Gelman2008]_ advocates to rescale numerical feature by two instead of one standard deviations and leave binary indicators untouched, 0 or 1, see parameter `with_std`.
```

Reference
---------

>     .. [Gelman2008] Gelman, A. (2008). Scaling regression inputs by 
> dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873

Reviewer 3: ```suggestion
    .. [Gelman2008] Gelman, A. (2008).
       :doi:`Scaling regression inputs by dividing by two standard deviations.
       <10.1002/SIM.3107>`. Statistics in medicine, 27(15), 2865-2873
```

        >>> print_options = np.get_printoptions()
        >>> np.set_printoptions(suppress=True)

> Scaled data has zero mean and unit or factor 2 standard deviation [Gelman2008]_.
> refernce to **Gelman, A. (2008). Scaling regression inputs by** 
> **dividing by two standard deviations. Statistics in medicine, 27(15)
> , 2865-2873**.::

Reviewer 3: ```suggestion
The scaled data `X_scaled` has zero mean and unit standard deviation::
```

        >>> print_options = np.get_printoptions()
        >>> np.set_printoptions(suppress=True)

> Scaled data has zero mean and unit/factor 2 standard deviation in

Reviewer: ```suggestion
Scaled data has zero mean and unit or factor 2 standard deviation [Gelman2008]_.
```
